frontend/packages/console-app/src/components/nodes/NodesPage.tsx

```diff
@@ -622,6 +642,23 @@ const getFilters = (t: TFunction): RowFilter<NodeRowItem>[] => [
       return input.selected?.some((r) => nodeRoles.includes(r));
     },
   },
+  {
+    filterGroupName: t('console-app~CPU architecture'),
+    type: 'cpu-arch',
+    reducer: getNodeArchitecture,
+    items: [
+      { id: 'amd64', title: t('console-app~amd64') },
+      { id: 'ppc64le', title: t('console-app~ppc64le') },
+      { id: 'arm64', title: t('console-app~arm64') },
+      { id: 's390x', title: t('console-app~s390x') },
+    ],
+    filter: (input, obj) => {
+      if (!input.selected?.length) {
+        return true;
+      }
+      return isCSRResource(obj) || input.selected?.includes(getNodeArchitecture(obj));
```

Comment:
We return 'amd64' from `getNodeArchitecture` when `isCSRResource` returns true. Is this accurate, or do we just not have an architecture for these resources? With the current logic, CSR nodes will always be shown, regardless of what filters are selected. Is this the desired behavior?

Reply:
`getNodeArchitecture` will return the arch based on the node arch, it can be ppc64le/amd64/arm64 . even with `isCSRResource` return True/False it will be in or condition , so the arch of the node will be displayed in the UI.

Reply:
This logic causes the filter function to always return `true` if the node is a CSR node. This means CSR nodes will always be shown, no matter what is selected in the filter options. Is this the desired behavior?

Reply:
This also ties into @spadgett's question about CSR needing a special case. Don't these nodes have an architecture as well?